api/src/main/java/org/openmrs/module/fhir/api/db/FHIRDAO.java

```diff
@@ -21,4 +27,8 @@
 	/*
 	 * Add DAO methods here
 	 */
+
+	<Ord extends Order> List<Ord> getOrdersByAccessionNumber(String accessionNumber);
+	//List<Obs> getObsForOrderId(Object orderId);
```

Comment:
Did you intentionally leave the above commented out? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode